src/mavrosflight/sensors/imu.cpp

```diff
@@ -1,28 +1,116 @@
 /**
  * \file imu.cpp
  * \author Daniel Koch <daniel.koch@byu.edu>
+ * \author James Jackson <me@jamessjackson.com>
  */
 
 #include <mavrosflight/sensors/imu.h>
+#include <ros/ros.h>
+#include <deque>
+#include <eigen3/Eigen/Core>
+#include <eigen3/Eigen/Dense>
 
 namespace mavrosflight
 {
 namespace sensors
 {
 
+Imu::Imu()
+{
+  x_[0] << 0, 0;
+  x_[1] << 0, 0;
+  x_[2] << 0, 0;
```

Comment:
Actually, I think, since this is already going to be on the naze, `fcu_io` doesn't need to know what these are unless it's doing a calibration.